mlc_llm/relax_model/minigpt.py

```diff
@@ -535,7 +534,7 @@ def get_model(args):
             raise ValueError(
                 "MiniGPT model path should be a single file instead of a directory."
             )
-        llama_state_dict = torch.load(model_path, map_location="cpu")["model"]
+        llama_state_dict = torch.load(model_path + ".pth", map_location="cpu")["model"]
```

Comment:
When `--model` is `minigpt4-7b` when running `build.py`, `model_path` here was `dist/minigpt4-7b`, which does not point to the pth file.

Reply:
cc @Kathryn-cat

Reply:
okie thanks!